keras_cv/layers/preprocessing/random_hue.py

```diff
@@ -28,36 +28,44 @@ class RandomHue(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     hue channel (H) by delta. The image is then converted back to RGB.
 
     Args:
-        factor: A tuple of two floats, a single float or `keras_cv.FactorSampler`.
-            factor` controls the extent to which the image saturation is impacted.
-            `factor`=`0.0`, `0.5` or `1.0` makes this layer perform a no-op operation.
-            `factor=0.25` and `factor=0.75` makes the image to have fully opposite
-            hue value. Values should be between `0.0` and `1.0`.
-            If a tuple is used, a `factor` is sampled
-            between the two values for every image augmented.  If a single float is
-            used, a value between `0.0` and the passed float is sampled.
-            In order to ensure the value is always the same, please pass a tuple with
-            two identical floats: `(0.5, 0.5)`.
+        factor: A tuple of two floats, a single float or a
+            `keras_cv.FactorSampler`.  `height_factor` controls the size of the
```

Comment:
I think this should be `factor`.

Reply:
And the whole docstring seems to be copied from other layer.

Reply:
Sorry about that - forgot to update the text.  updated.